src/Mailgun/Constants/ExceptionMessages.php

```diff
@@ -18,7 +18,7 @@ class ExceptionMessages
     const EXCEPTION_GENERIC_HTTP_ERROR = 'An HTTP Error has occurred! Check your network connection and try again.';
     const EXCEPTION_MISSING_REQUIRED_PARAMETERS = 'The parameters passed to the API were invalid. Check your inputs!';
     const EXCEPTION_MISSING_REQUIRED_MIME_PARAMETERS = 'The parameters passed to the API were invalid. Check your inputs!';
-    const EXCEPTION_MISSING_ENDPOINT = "The endpoint you've tried to access does not exist. Check your URL.";
+    const EXCEPTION_MISSING_ENDPOINT = "The endpoint you've tried to access does not exist. Check if the domain matches the domain you have configure on mailgun.";
```

Comment:
In terms of Code-Style: How do you usually pass parameters/configs (like `url` now) to other parts of the code? Because from what I see so far, I would need to get it from 1->2->3 to get add it to the Exception Output.

1. https://github.com/mailgun/mailgun-php/blob/master/src/Mailgun/Api/Message.php#L52
2. https://github.com/mailgun/mailgun-php/blob/master/src/Mailgun/Api/HttpApi.php#L72-L74
3. https://github.com/mailgun/mailgun-php/blob/master/src/Mailgun/Exception/HttpClientException.php#L71-L74